File: C# Advanced/03. Methods/05. Larger than neighbours/05. Larger than neighbours.cs
using System;
class Program
{
    static int Larger(int[] arr, int i)
    {
        if ((i > 0 && i < arr.Length -1) && 
           ((arr[i] > arr[i-1]) && (arr[i] > arr[i + 1])))
        {
            return 1;
        }
        else
        {
            return 0;
        }

    }
    static void Main()
    {
        int size = int.Parse(Console.ReadLine());
        string[] stringMatrix = Console.ReadLine().Split();
        int count = 0;
        int[] matrix = new int[size];
        for (int i = 0; i < size; i++)
        {
            matrix[i] = int.Parse(stringMatrix[i]);
        }
        for (int i = 0; i < matrix.Length; i++)
        {
            count += Larger(matrix, i);
        }
        Console.WriteLine(count);
    }
}

File: JS/08. Functions/05. Larger than neighbours.js
function solve(args) {
    'use strict';
    let arr = args[1].split(' '),
        l = arr.length,
        count = 0;

    for (let i = 1; i < l - 1; i += 1) {
        if (largestThan(i, arr[i])) {
            count++;
        }
    }
    console.log(count);

    function largestThan(j, element) {
        if ((+element > +arr[j - 1]) && (+element > +arr[j + 1])) {
            return true;
        } else {
            return false;
        }
    }
}
solve(['8', '-26 -25 -28 31 2 27']);